Description: Disable configure parameters on phpinfo() output
 .
 Patch needs to be discussed with upstream and the issues that lead to
 its addition re-checked.  Quoting changelog entry:
 .
 Add [...], which disables the display of our "Configure Command" in
 phpinfo(), which was the source of many bogus bug reports over the
 years, due to people misinterpreting its meaning.
Origin: vendor
Forwarded: no
Last-Update: 2010-01-18

--- a/ext/standard/info.c
+++ b/ext/standard/info.c
@@ -704,7 +704,7 @@ PHPAPI void php_print_info(int flag TSRM
 #ifdef ARCHITECTURE
 		php_info_print_table_row(2, "Architecture", ARCHITECTURE);
 #endif
-#ifdef CONFIGURE_COMMAND
+#if 0
 		php_info_print_table_row(2, "Configure Command", CONFIGURE_COMMAND );
 #endif
 
--- a/ext/standard/tests/general_functions/phpinfo.phpt
+++ b/ext/standard/tests/general_functions/phpinfo.phpt
@@ -20,7 +20,6 @@ PHP Version => %s
 
 System => %s
 Build Date => %s%a
-Configure Command => %s
 Server API => Command Line Interface
 Virtual Directory Support => %s
 Configuration File (php.ini) Path => %s
